assets/training/distillation/components/pipeline_validation/spec.yaml

```diff
@@ -0,0 +1,147 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: oss_distillation_validate_pipeline
+version: 0.0.1
+type: command
+
+is_deterministic: true
+
+display_name: OSS Distillation Validate Pipeline
+description: Component to validate inputs to the distillation pipeline
+
+environment: azureml://registries/azureml/environments/acft-hf-nlp-gpu/versions/63
```

Comment:
v63 is pretty old and will hit eol soon, also some vulns concerns with the older version

Reply:
@vizhur - I've bumped this up to the latest version now :)